python/README.md

```diff
@@ -27,15 +29,49 @@ pip install mujoco
 A copy of the MuJoCo library is provided as part of the package and does **not**
 need to be downloaded or installed separately.
 
-If you wish to modify and build the bindings from source, you should clone the
-entire `mujoco` repository from GitHub, then run the `make_sdist.sh` script to
-generate a [source distribution](https://packaging.python.org/en/latest/glossary/#term-Source-Distribution-or-sdist)
-tarball, then run `pip wheel name_of_sdist.tar.gz`. The `make_sdist.sh` script
-generates additional C++ header files that are needed to build the bindings,
-and also pulls in required files from elsewhere in the repository outside the
-`python` directory into the sdist.
+### Source
+
+**Note.** Building from source is only necessary if you are modifying the Python bindings (or are trying to run on exceptionally old Linux systems). If that's not the case, then we recommend installing the prebuilt binaries from PyPI.
+
+Before proceeding, make sure you have CMake and a C++17 compiler installed.
+
+1. Download the latest release of the [binary archives](https://github.com/deepmind/mujoco/releases) from GitHub. On macOS, the download corresponds to a DMG file from which you can drag the `MuJoCo` app into your Applications folder.
+
+<p align="center">
+  <img src="../assets/macos_install.png" width=600>
+</p>
+
+2. Clone the entire `mujoco` repository from GitHub and cd into the python directory.
+
+```bash
+git clone https://github.com/deepmind/mujoco.git
+cd mujoco/python
+```
+
+3. Create a virtual environment:
+
+```bash
+python3 -m venv /tmp/mujoco && /tmp/mujoco/bin/activate
```

Comment:
Should be `&& source /tmp/mujoco/bin/activate`. My bad.

May be clearer to just put the second command it in a separate line.